src/kemal/log_handler.cr

```diff
@@ -1,23 +1,22 @@
 module Kemal
   # Uses `STDOUT` by default and handles the logging of request/response process time.
-  class LogHandler < Kemal::BaseLogHandler
```

Comment:
Why was the inheritance broken here?

Reply:
It wasn't needed anymore because it logs to `Log` instead of `@io`. However maybe would be better to create a new class `RequestLogHandler` that logs using Log and keep this `LogHandler` class as is for backwards compatibility, what do you think?